README.md

```diff
@@ -32,6 +35,7 @@ PostgreSQL versions currently supported are:
 * [postgresql-10](https://github.com/sclorg/postgresql-container/tree/generated/10)
 * [postgresql-12](https://github.com/sclorg/postgresql-container/tree/generated/12)
 * [postgresql-13](https://github.com/sclorg/postgresql-container/tree/generated/13)
+* [postgresql-15](https://github.com/sclorg/postgresql-container/tree/generated/15)
```

Comment:
As far as I know. This is not needed. We have already a version as a part of the upstream repo. Like for version 13 (https://github.com/sclorg/postgresql-container/tree/master/13).

Reply:
Good point, will fix.